core/Piwik.php

```diff
@@ -572,7 +572,8 @@ public static function isValidEmailString($emailAddress)
     {
         /** @var \Zend_Validate_EmailAddress $zendEmailValidator */
         $zendEmailValidator = StaticContainer::get('Zend_Validate_EmailAddress');
-        return $zendEmailValidator->isValid($emailAddress);
+
+	return $zendEmailValidator->isValid(mb_convert_encoding($emailAddress, "UTF-8", "HTML-ENTITIES"));
```

Comment:
There's a core Piwik function for unsanitizing values: `\Piwik\Common:: unsanitizeInputValue(...).` (see https://github.com/matomo-org/matomo/blob/3.x-dev/core/Common.php#L363 ).

Can you use that instead of `mb_convert_encoding()`? Depending on a function we control means we can change how it works ourselves & in one place, instead of having to look for every use of, eg, `mb_convert_encoding()`.